template/data/fr/global.json

```diff
@@ -0,0 +1,116 @@
+{
+  "globalFR": {
+    "id": 2,
+    "locale": "fr",
+    "metaTitleSuffix": "Strapi Corporate",
+    "metadata": {
+      "metaTitle": "Strapi starter for Corporate Sites",
+      "metaDescription": "Créer votre site corporate avec Strapi",
+      "twitterCardType": "summary",
+      "twitterUsername": "strapijs",
+      "shareImage": null
+    },
+    "notificationBanner": {
+      "text": "Cette page a été crée avec le Strapi starter pour les sites Corporate.\n [Voir les autres Strapi starters](https://strapi.io/starters)",
+      "type": "info"
+      
+    },
+    "navbar": {
+      "links": [
+        {
+          "url": "/tarifs",
+          "newTab": false,
+          "text": "Tarifs"
+        },
+        {
+          "url": "/nous-contacter",
+          "newTab": false,
+          "text": "Nous Contacter"
```

Comment:
I think if we want to capitalize all links, it should rather be a CSS rule on the frontend